### ISSUE TITLE
Check ple.27.7e symbols 

### ISSUE THREAD

**Issue by odontaster** [slarge/fisheryO#54]
Last years F is red, but the MSFD indicator appear green

**Comment by sarahlmillar** [comment]
There a mistake here. DC32 and GES in the annex table should be red. Not sure why its wrong - its correct in SAG?

**Comment by odontaster** [comment]
Yes, probably some issue with the fact that is "proxy" will keep an eye on this. Now fixed in the table.

**Comment by odontaster** [comment]
sorry again, I think is D3C1 that should be red, not D3C2. Please check and confirm.

**Comment by sarahlmillar** [comment]
yes, D3C1 should be red you are right

**Comment by odontaster** [comment]
done
